src/logic/safe/store/actions/fetchSafe.ts

```diff
@@ -128,14 +128,14 @@ export const checkAndUpdateSafe = (safeAdd: string) => async (dispatch: Dispatch
 
   // If the remote owners does not contain a local address, we remove that local owner
   localOwners.forEach((localAddress) => {
-    const remoteOwnerIndex = remoteOwners.findIndex((remoteAddress) => sameAddress(remoteAddress, localAddress))
+    const remoteOwnerIndex = remoteOwners?.findIndex((remoteAddress) => sameAddress(remoteAddress, localAddress))
```

Comment:
@matextrem can you add a comment here why `remoteOwners` can be null or undefined?

Reply:
I had a bug in batchRequest function. Fixed now.